northd/northd.c

```diff
@@ -9123,8 +9173,19 @@ build_lswitch_ip_unicast_lookup(struct ovn_port *op,
                     continue;
                 }
                 ds_clear(match);
-                ds_put_format(match, "eth.dst == "ETH_ADDR_FMT,
+                ds_put_format(match, "eth.dst == { "ETH_ADDR_FMT,
                               ETH_ADDR_ARGS(mac));
+                struct lport_addresses proxy_arp_addrs;
```

Comment:
This one slipped through, but obviously it also needs to be freed `destroy_lport_addresses`.